index.html

```diff
@@ -0,0 +1,378 @@
+	L7.1,8.5l1.4-1.4l3.5,3.5l3.5-3.5l1.4,1.4L13.4,12L16.9,15.5z"/>
+                <rect class="st0" width="24" height="24"/>
+                </svg>
+            </li>
+          </ul>
+
+        </nav>
+        <a href="#" class="request-demo">
+          Request Demo
+          <svg
+            width="22"
+            height="16"
+            viewBox="0 0 22 16"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path
+              d="M15 15L21 8L15 1"
+              stroke-width="2"
+              stroke-linecap="round"
+              stroke-linejoin="round"
+            />
+            <path
+              d="M21 8H1"
+              stroke-width="2"
+              stroke-linecap="round"
+              stroke-linejoin="round"
+            />
+          </svg>
+        </a>
+      </div>
+    </header>
+    <main class="main">
+      <div class="wrapper main__wrapper">
+        <section class="who-we-are">
+          <div class="who-we-are__first-div">
+            <img src="src/images/who-we-are_bg.svg" alt="" />
+            <h1 class="who-we-are__title">Save your data storage here.</h1>
+            <p class="who-we-are__text">
+              Data Warehouse is a data storage area that has been tested for
+              security, so you can store your data here safely but not be afraid
+              of being stolen by others.
+            </p>
+            <a class="who-we-are_link" href="#"> Learn more </a>
+
+          </div>
+          <div class="who-we-are__second-div">
+            <img src="src/images/who-we-are_bg2.svg" alt="" />
+            <div>
+              <h2 class="subtitle who-we-are__subtitle">
+                We are a high-level data storage bank
+              </h2>
+              <div class="who-we-are__text">
+                <p>
+                  The place to store various data that you can access at any
+                  time through the internet and where you can carry it.
+                </p>
+                <p>
+                  This very flexible storage area has a high level of security.
+                  To enter into your own data you must enter the password that
+                  you created when you registered in this Data Warehouse.
+                </p>
+              </div>
+            </div>
+          </div>
+        </section>
+        <section class="features">
+          <h2 class="subtitle feature__subtitle">Features</h2>
+          <p class="feature__text">
+            Some of the features and advantages that we provide for those of you
+            who store data in this Data Warehouse.
+          </p>
+          <div class="cards">
+            <div class="card card1">
+              <div class="card__info">
+                <h3 class="card__title">Search Data</h3>
+                <p class="card__text">
+                  Don’t worry if your data is very large, the Data Warehoue
```

Comment:
The character U+2019 "’" could be confused with the character U+0060 "`", which is more common in source code. [Adjust settings]